db/version_set.h

```diff
@@ -982,7 +982,8 @@ class VersionSet {
   // in levels [start_level, end_level). If end_level == 0 it will search
   // through all non-empty levels
   uint64_t ApproximateSize(Version* v, const Slice& start, const Slice& end,
-                           int start_level = 0, int end_level = -1);
+                           int start_level = 0, int end_level = -1,
+                           bool for_compaction = false);
```

Comment:
Not a big fan of default value for function arguments.

Reply:
I removed the default values.